docs/k8s/run.sh

```diff
@@ -82,4 +87,7 @@ if [[ ! -z "${TS_DEST_IP}" ]]; then
   iptables -t nat -I PREROUTING -d "$(tailscale --socket=/tmp/tailscaled.sock ip -4)" -j DNAT --to-destination "${TS_DEST_IP}"
 fi
 
-fg
+trap handler SIGINT SIGTERM
```

Comment:
Put this immediately after PID=... so that if I ^C during startup, it still all works out?

Reply:
good call! done